lab-justin/greet.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
```

Comment:
Great Use of Project Structure ...
seporating your code into **lib** and **test** directorys can make a world of difference when trying to navigate your code